react/FileSelector/Readme.md

```diff
@@ -0,0 +1,48 @@
+This component is used to display a customizable file selector.
+
+### Default
+
+```jsx
+<FileSelector
+  renderContent={() => <span>Click me to choose file</span>}
+  processSelected={console.log}
```

Comment:
Again, we're just surcharging the native `onChange`. I think `onChange` or `onSelected` are more accurate.

Reply:
:+1: